examples/reason-react-example/src/basics/Tuples.re

```diff
@@ -0,0 +1,48 @@
+open Belt;
```

Comment:
Should I add Tuples.re also in the TypeScript folder?

Reply:
See below.